ISISScientificComputing/autoreduce
# Python 3 - Build Steps

### Summary of work
* Fix setup.py database
* Fix setup.py start

### How to test your work
* Ensure that travis passes the setup.py build steps without failure

Fixes #406
